core/geometry/src/polyface/TaggedNumericData.ts

```diff
@@ -0,0 +1,167 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Geometry } from "../Geometry";
+
+/** @packageDocumentation
+ * @module Polyface
+ */
+
+/**
+ * `TaggedNumericConstants` defines enums with constant values for use in tags of `TaggedNumericData`

```

Comment:
Can you use doc links here and elsewhere so people don't have to search to find out what "TaggedNumericData" is?
`[[TaggedNumericData]]`

Reply:
Ah, that's easy.  Tx.